mediator/test/e2e-test.sh

```diff
@@ -26,11 +31,11 @@ cd $BASEDIR
 ./startup.sh destroy
 
 echo 'Starting the interoperability containers...'
-cd $BASEDIR
+./startup.sh up-test
 retry_startup
 
 echo 'Waiting for configurator to finish...'
-docker container wait chis-interop-configurator-1
+docker container wait chis-interop-cht-configurator-1
```

Comment:
The container name should be: `chis-interop-configurator-1`
As [this](https://github.com/medic/cht-interoperability/actions/runs/11478501793/job/31942842516) is failing.

Reply:
chis-interop-configurator and interop-cht-configurator are two different containers. chis-interop-configurator sets up openhim configuration, chis-interop-cht-configurator runs cht-conf to add the cht config for e2e-tests

I guess it should be waiting for both here

Reply:
But I don't see `interop-cht-configurator` creation in the logs.

Reply:
Same here, I don't see an `interop-cht-configurator` container. @witash are we missing anything?

Reply:
When running the test locally, there is an `interop-cht-configurator.` In CI the behavior is different, maybe due to the [too](toomanyrequests: Rate exceede) problem. 

Locally the OpenMRS tests are failing :
```
Workflows
    OpenMRS workflow
      ✕ Should follow the CHT Patient to OpenMRS workflow (4710 ms)
      ✕ Should follow the OpenMRS Patient to CHT workflow (204 ms)
    Loss To Follow-Up (LTFU) workflow
      ✓ Should follow the LTFU workflow (8371 ms)

● Workflows › OpenMRS workflow › Should follow the CHT Patient to OpenMRS workflow
> 140 |       expect(retrieveFhirPatientIdResponse.body.total).toBe(1);

  ● Workflows › OpenMRS workflow › Should follow the OpenMRS Patient to CHT workflow
> 177 |       expect(retrieveFhirPatientIdResponse.body.total).toBe(1);
```

Both asserting that the patiend ID was created in Fhir.
I'm testing this manually to see if I can reproduce it.